### PR TITLE
NEWS: Mention notable changes for Ultima 8

### PR DESCRIPTION
**Pull request by OMGPizzaGuy**
I figured news should mention the user-noticeable changes since last release.
Are these good entries?

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -135,6 +135,11 @@ For a more comprehensive changelog of the latest experimental code, see:
    - Fixed recenter the screen on activating an inventory item.
    - Fixed pressing W to talk to people also makes you jump.
 
+ Ultima8:
+   - Added support for saving and resizing of minimap.
+   - Adjust cursor to behave closer to the original.
+   - Adjust quantity slider to behave closer to the original.
```
Comment (bluegr) [inline NEWS.md:141]: Can you provide more information regarding the functionality of the quantity slider?

Comment (OMGPizzaGuy) [reply NEWS.md:141]: Would "item quantity slider" be better?

Comment (bluegr) [reply NEWS.md:141]: > Would "item quantity slider" be better?

Yes, that sounds better :)

**Comment by bluegr** [comment]
Thanks!

**Pull request merged**